infrastructure/src/test/java/org/corfudb/infrastructure/SequencerServerTest.java

```diff
@@ -323,9 +323,9 @@ public void testBootStrapSequencer() {
         long globalTailA = 0;
         long globalTailB = 1;
         StreamAddressSpace streamAddressSpaceA = new StreamAddressSpace(Address.NON_ADDRESS,
-                Roaring64NavigableMap.bitmapOf(globalTailA));
-        StreamAddressSpace streamAddressSpaceB = new StreamAddressSpace(Address.ABORTED,
```

Comment:
@zfrenette This is a bug in the unit test, right? (Aborted shouldnt be used here)

Reply:
Yeah, `ABORTED` should not have been used... `NON_ADDRESS` is more appropriate here.